github/VisualStudio
# Stop Blend from crashing when Team Explorer window is opened

When GitHub for Visual Studio is installed for all users, it installs into Visual Studio, the experimental instance of Visual Studio and Expression Blend (now that Blend is based on the Visual Studio shell). 

Expression Blend seems to support MEF but has limited package functionality. It doesn't seem to support custom services, loading a package but its guid or menu commands. This breaks various GitHub for Visual Studio MEF exports that attempt to call custom services.

This was crashing Expression Blend when the Team Explorer window was opened (which could lead to crashes on launch 😭).

To test this fix:

1. Install an old version of the extension.
2. Open Blend and the Team Explorer window.
3. Expect Blend to crash.
4. Install a build from this PR (you need to build a non-experimental/allusers .vsix).
5. Open Blend and the Team Explorer window.
6. Expect GitHub for Visual Studio to no longer be visible.

NOTE: There will still be a GitHub option on `Connect > Manage Connections` but it won't be functional. Let me know if you find it anywhere else of if it can be made to crash.

Fixes #951
